README.md

```diff
@@ -75,21 +81,29 @@ def fetchString(n: Int): Fetch[String] = Fetch(n) // or, more explicitly: Fetch(
 Now that we can convert `Int` values to `Fetch[String]`, let's try creating a fetch.
 
 ```scala
-import fetch.implicits._
 import fetch.syntax._
 
 val fetchOne: Fetch[String] = fetchString(1)
 ```
 
-Now that we have created a fetch, we can run it to a target monad. Note that the target monad (`Eval` in our example) needs to implement `MonadError[M, Throwable]`, we provide an instance for `Eval` in `fetch.implicits._`, that's why we imported it.
+We'll run our fetches to the ambien `Id` monad in our examples, let's do some imports.
 
 ```scala
-val result: String = fetchOne.runA[Eval].value
-// ToStringSource OneAnd(1,List())
-// result: String = 1
+import cats.Id
+import fetch.unsafe.implicits._
+import fetch.syntax._
 ```
 
-As you can see in the previous example, the `ToStringSource` is queried once to get the value of 1.
+Note that in real-life scenarios you'll want to run a fetch to a concurrency monad such as `Future` or `Task`, synchronous execution of a fetch
```

Comment:
I think this should move up to underneath the previous paragraph, that way all the code can lice together.